README.md

```diff
@@ -714,6 +714,8 @@ Table of Contents
    * [hotjar.com](https://www.hotjar.com/) — Per site: 2,000 pages views/day, 3 heatmaps, data stored for 3 months,...
    * [usersurge.com](https://www.usersurge.com/) — 250K sessions per month for individuals.
    * [smartlook.com](https://www.smartlook.com/) — free packages for web and mobile apps (1500 sessions/month), 3 heatmaps, 1 funnel, 1 month data history
+   * [mouseflow.com](https://mouseflow.com/) — 100 sessions/month free for 1 website
+   
```

Comment:
please do not add more whitespace here - and remove the already extra one if you dont mind :)

Reply:
removed unnecessary line breaks.